src/metabase/search/appdb/specialization/postgres.clj

```diff
@@ -29,12 +29,11 @@
   (when (seq entries)
     (t2/query
      ;; The cost of dynamically calculating these keys should be small compared to the IO cost, so unoptimized.
-     (let [update-keys (vec (disj (set (keys (first entries))) :id :model :model_id))
-           excluded-kw (fn [column] (keyword (str "excluded." (name column))))]
+     (let [update-keys (vec (disj (set (keys (first entries))) :id :model :model_id))]
```

Comment:
Ah, I see this is where you got some of that horrible transformation I complained about from - curse you Past Chris!

I feel like doing a `(vec (remove #{:id ...} ...)` is not only more efficient but also much clearer.